doc/source/installation.rst

```diff
@@ -47,7 +47,35 @@ In addition
 
 are used to build and test the documentation.
 
-A few other packages such as LaTeX is used for circuit plotting, please refer to the main documentation section for detailed instruction.
+.. _circuit_plot_packages:
+
+Plotting Circuits
+------------------
+In order to plot circuits, following packages are needed - ``LaTex``, ``ImageMagick``,
+``QCircuit``, ``pdfcrop`` and ``pdflatex``.
+
+**For Linux** : If you would prefer to avoid installing the full ``texlive``
+package for ``LaTex``, `this link has <https://tex.stackexchange.com/a/504566/203959>`_
+some useful discussion on selectively installing smaller packages. ``LaTex`` is needed
+to plot the circuits.
+
+* ``braket`` will be installed as a part of ``texlive-latex-extra``
+* ``qcircuit`` will be installed as a part of ``texlive-pictures``
+* ``pdfcrop`` and ``pdflatex`` are installed when a minimal ``texlive`` is installed.
+
+``ImageMagick`` can be installed as a conda package via `this link <https://github.com/conda-forge/imagemagick-feedstock#installing-imagemagick>`_
+or use this `link <https://imagemagick.org/script/download.php>`_ to install
+via source. This package along with ``pdfcrop`` and ``pdflatex`` are
+needed to display the circuit diagrams.
+
+
+You `might need to make changes <https://stackoverflow.com/a/52863413/10241324>`_ to ``policy.xml`` if following error occurs :
+
+.. code-block:: text
+
+  RuntimeError: convert-im6.q16: not authorized `qcirc.pdf' @ error/constitute.c/ReadImage/412.
+  convert-im6.q16: no images defined `qcirc.png' @ error/convert.c/ConvertImageCommand/3258.
+
```

Comment:
A few questions/comments:
- `pdflatex` is not a separate package is it? It is part of LaTeX.
- The official ImageMagick installer is not from source, it is a binary distribution.
- Let's not recommend `conda install imagemagick`. I remember neither you nor I succeeded on this, right?

Reply:
> pdflatex is not a separate package is it? It is part of LaTeX.

Yes, I haven't pushed the new changes yet. I have made a note that they should not install `pdflatex` via pip. I made the mistake and I think that guy also had a similar issue. 

>  Let's not recommend conda install imagemagick. I remember neither you nor I succeeded on this, right?

I will have to double check how it worked for me. I tried way too many things and forgot how I installed it. I think `apt-get` might have worked for me. 

> The official ImageMagick installer is not from source, it is a binary distribution.

Thanks ! I linked [the wrong page](https://imagemagick.org/script/install-source.php). 

> We also need Ghostscript for the pdf to png conversion.

I don't think I had to install it separately. I can check which package installed it as a dependency.

Reply:
> I don't think I had to install it separately. I can check which package installed it as a dependency.

Ok, I remember I did `apt-get ghostscript`.

> I will have to double check how it worked for me. I tried way too many things and forgot how I installed it. I think apt-get might have worked for me.

Sure, just if it takes too much time, there is no need to put it on high priority. We can slowly work this out. The decomposition has higher priority ;)

Reply:
> there is no need to put it on high priority. We can slowly work this out

I have linked source installation instructions for now. We can update the instructions later about using `apt-get` if someone runs into new issues.